management/src/main/java/io/micronaut/management/health/indicator/Readiness.java

```diff
@@ -0,0 +1,17 @@
+package io.micronaut.management.health.indicator;
```

Comment:
put these in a subpackage called `io.micronaut.management.health.indicator.annotation`